pkg/cmd/util.go

```diff
@@ -82,23 +88,27 @@ func getApplicationsFromFiles(files []string) ([]inputData, error) {
 func GetAllYAMLFiles(paths []string) ([]string, error) {
 	var files []string
 	for _, path := range paths {
-		fileInfo, err := os.Stat(path)
-		if err != nil {
-			return nil, errors.Wrapf(err, "can't get file info about %s", path)
-		}
-		if fileInfo.IsDir() {
-			ymlFiles, err := filepath.Glob(filepath.Join(path, "*.yml"))
+		if path != "-" {
+			fileInfo, err := os.Stat(path)
 			if err != nil {
-				return nil, errors.Wrapf(err, "can't list *.yml files in %s", path)
+				return nil, errors.Wrapf(err, "can't get file info about %s", path)
```

Comment:
`can't retrieve file information about`

Reply:
github is messed up here, I didn't change anything

Reply:
I know, but the correct grammar is `can't retrieve file information about %s`